cmd/booster-bitswap/filters/filters.go

```diff
@@ -131,13 +131,19 @@ func NewMultiFilter(
 	requestCounter RequestCounter,
 	apiFilterEndpoint string,
 	apiFilterAuth string,
+	customBadBitsDenyList string,
 ) *MultiFilter {
 	filters := []FilterDefinition{
 		{
 			CacheFile: filepath.Join(cfgDir, "denylist.json"),
 			Fetcher:   FetcherForHTTPEndpoint(BadBitsDenyList, ""),
 			Handler:   NewBlockFilter(),
 		},
+		{
+			CacheFile: filepath.Join(cfgDir, "customdenylist.json"),
+			Fetcher:   FetcherForHTTPEndpoint(customBadBitsDenyList, ""),
+			Handler:   NewBlockFilter(),
+		},
```

Comment:
can this just be 1 deny list, where it defaults to the canonical `BadBits` location when not over-ridden?

Reply:
@willscott We don't want to override the default one. I would prefer to keep the default one always on and deny additional ones based on the SP's input. This would be useful in cases SPs themselves have identified bad content. The default Badbists list is already verified and should always be denied to avoid copyright issues.

Reply:
* agree we want SPs to be able to identify bad content and block it.
* can the flag for 'bad bits' that can be configured be an array of urls? when not specified it can be defaulted to `[OfficialBadBits]`
* The ability to override the default one is important for decentralization, "you don't have to rely on centralized infrastructure to be an SP"

Reply:
That makes sense. Let me redesign with the ability to override the default list if required.

Reply:
huh I totally never considered the option of multiple filters but the multifilter would totally support that. But I agree, I'd rather it  be a replacable array rather than a single optional add.

Reply:
and I don't like initializing a filter in the absense of a URL